### PR TITLE
Typography: Added line height support

### PR DESCRIPTION
**Pull request by Ydus**
### Changes

- Added line height support for typographies (copied from the internal implementation)

### CODE REVIEW

**Comment by JiriTrecak** [comment]
@chillbrodev Merged the changes :) I'll update it for everyone on Supernova Cloud

**Comment by JiriTrecak** [comment]
@chillbrodev Thank you for reminding us :) It has been updated for everyone, as long as you are using 1.2.2 on SN Cloud it should all work, I tested it for myself and seemed all good

**Comment by chillbrodev** [comment]
@JiriTrecak Thanks for this! Much appreciated

File: src/exported_files/text_styles.pr [hunk 1]
```diff
@@ -38,6 +46,10 @@ class AppTextStyles {
     fontSize: {[ inject "export_measure_value" context typographyToken.value.fontSize /]},
     decoration: TextDecoration.{{ decoration }},
     letterSpacing: {{ letterSpacing.toString() }},
+    {[ if lineHeight != "null" ]}
+    height: {{ lineHeight.toString() }},
+    {[/]}
+    textStyle: const TextStyle(leadingDistribution: TextLeadingDistribution.even),
```
Comment (chillbrodev) [inline src/exported_files/text_styles.pr:52]: @JiriTrecak So this `textStyle` property is being added but is not supported by `TextStyle`. A nested TextStyle in a TextStyle doesn't work. Therefore this breaks the importer such that it requires manual input to remove `textStyle` and replace it with `leadingDistribution: TextLeadingDistribution.even`. 

This is the error message:
```
	"message": "The named parameter 'textStyle' isn't defined. Try correcting the name to an existing named parameter's name, or defining a named parameter with the name 'textStyle'.",
```

**Comment by JiriTrecak** [comment]
@chillbrodev This has been fixed with another PR, and merged already. LMK if something still doesn't work! :)

**Pull request merged**